### PR TITLE
landing page redirect

### LINKED ISSUE #242: Ensure Logged-In Users cannot access landing page
Currently, when a user logs in from the landing page, they are redirected to the dashboard. However, logged-in users can still access the landing page by navigating back manually or typing the URL. This feature should ensure that once a user is logged in, they cannot access the landing page and are always redirected to the dashboard. This is a fullstack issue not just a frontend one

### PR DESCRIPTION
**Pull request by Xaxxoo**
## Detailed Information

---

## Related Issues

Closes #242 

---

## Type of Change

- [ ] 🐛 Bug fix or ⚙️ enhancement
- [ ] ✨ New feature or Chore (change with no new features or fixes)
- [ ] 📚 Documentation update

---

## Checklist (select as many as applicable)

- [ ] The code follows the style guidelines of this project.
- [ ] All new and existing tests pass.
- [ ] This pull request is ready to be merged and reviewed.

### CODE REVIEW

**Review by Solomonsolomonsolomon — COMMENTED** [review]
@Xaxxoo you included node_modules, please remove

**Comment by Xaxxoo** [comment]
> @Xaxxoo you included node_modules, please remove

removed. My bad.

**Comment by fishonamos** [comment]
Please add the issue number to the description.

File: package.json [hunk 1]
```diff
@@ -2,6 +2,13 @@
   "dependencies": {
     "@apibara/indexer": "^0.4.1",
     "@apibara/protocol": "^0.4.9",
+    "dotenv": "^16.4.7",
```
Comment (Solomonsolomonsolomon) [inline package.json:5]: you're installing packages in the wrong directory

**Review by Solomonsolomonsolomon — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (Solomonsolomonsolomon) [inline pnpm-lock.yaml:1]: this is at the root folder,it is /app/client that uses a pnpm lock

File: package.json
Comment (Solomonsolomonsolomon) [inline package.json:1]: youre installing deps in the root folder, this should be in the /land-registry-backend

File: land-registry-backend/package-lock.json
Comment (Solomonsolomonsolomon) [inline land-registry-backend/package-lock.json:1]: please restore file as land-registry-backend uses npm. its /app/client that uses pnpm

File: app/client/package-lock.json
Comment (Solomonsolomonsolomon) [inline app/client/package-lock.json:1]: app/client still has a package-lock. it should use a pnpm-lock

**Review by Solomonsolomonsolomon — CHANGES_REQUESTED** [review]

**Pull request merged**